src/app/api/services/workflow/workflow.service.spec.ts

```diff
@@ -41,13 +41,61 @@ describe('WorkflowService', () => {
         expect(this.lastConnection.request.url).toContain('v1/workflow/schemes');
     });
 
-    it('should get workflows for a page', () => {
-        this.workflowService.getPageWorkflows('123').subscribe((workflows: Workflow[]) => {
-            expect(workflows).toEqual([
-                { name: 'Workflow 1', id: 'one' },
-                { name: 'Workflow 2', id: 'two' },
-                { name: 'Workflow 3', id: 'three' }
-            ]);
+    it('should get workflows page actions', () => {
+        let result;
+        const inode = 'cc2cdf9c-a20d-4862-9454-2a76c1132123';
+        this.workflowService.getPageWorkflows(inode).subscribe((res) => {
+            result = res;
         });
+
+        this.lastConnection.mockRespond(
+            new Response(
+                new ResponseOptions({
+                    body: {
+                        entity: [
+                            {
+                                assignable: true,
+                                commentable: true,
+                                condition: '',
+                                icon: 'workflowIcon',
+                                id: '44d4d4cd-c812-49db-adb1-1030be73e69a',
+                                name: 'Assign Workflow',
+                                nextAssign: 'db0d2bca-5da5-4c18-b5d7-87f02ba58eb6',
+                                nextStep: '43e16aac-5799-46d0-945c-83753af39426',
+                                nextStepCurrentStep: false,
+                                order: 0,
+                                owner: null,
+                                requiresCheckout: true,
+                                roleHierarchyForAssign: true,
+                                schemeId: '85c1515c-c4f3-463c-bac2-860b8fcacc34',
+                                showOn: ['UNLOCKED', 'LOCKED'],
+                                stepId: null
+                            }
+                        ]
+                    }
+                })
+            )
+        );
+        expect(result).toEqual([
+            {
+                assignable: true,
+                commentable: true,
+                condition: '',
+                icon: 'workflowIcon',
+                id: '44d4d4cd-c812-49db-adb1-1030be73e69a',
+                name: 'Assign Workflow',
+                nextAssign: 'db0d2bca-5da5-4c18-b5d7-87f02ba58eb6',
+                nextStep: '43e16aac-5799-46d0-945c-83753af39426',
+                nextStepCurrentStep: false,
+                order: 0,
+                owner: null,
+                requiresCheckout: true,
+                roleHierarchyForAssign: true,
+                schemeId: '85c1515c-c4f3-463c-bac2-860b8fcacc34',
+                showOn: ['UNLOCKED', 'LOCKED'],
+                stepId: null
+            }
+        ]);
+        expect(this.lastConnection.request.url).toContain(`v1/workflow/contentlet/${inode}/actions`);
     });
```

Comment:
We have to have a error into the get method's test, because it should fail with this change

https://github.com/dotCMS/core-web/pull/567/files#diff-19e7868718b95e25be16d0be6b73900eR22